src/main/scala/temple/generate/utils/CodeTerm.scala

```diff
@@ -106,9 +106,12 @@ object CodeTerm {
     /** Add a string before the opening symbol, without any spacing */
     def prefix(prefix: String): CodeWrap = new CodeWrap(prefix + start, end)
 
-    /** Wrap a (comma-separated list of) terms in parentheses */
+    /** Wrap a (space-separated list of) terms in parentheses */
     def apply(string: CodeTerm*): String = mkCode(start, string, end)
 
+    /** Wrap a (comma-separated list of) terms in parentheses */
+    def list(string: CodeTerm*): String = mkCode(start, mkCode.list(string), end)
```

Comment:
boo now I got to make changes again

Reply:
Should also probs mention this stuff in PR desc, since it's not mentioned in title

Reply:
Sorry, noticed it was missing when I saw the comment on apply was wrong